packages/material/scss/table/_variables.scss

```diff
@@ -30,12 +30,12 @@ $kendo-table-cell-padding-x: null !default;
 $kendo-table-cell-padding-y: null !default;
 
 $kendo-table-sm-cell-padding-x: k-map-get( $kendo-spacing, 2.5 ) !default;
```

Comment:
There are still some values left, not using `k-spacing( n )` function.
Please search e.g `$kendo-spacing,` globally at your local repo and replace them where necessary.

**Filter:**
![image](https://github.com/telerik/kendo-themes/assets/138570637/410bc2ee-5a4b-4a5c-82e2-fb1f971965c9)

Reply:
Not all of them should be replaced, however I replaced where needed :)